spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -23,6 +23,14 @@ link:src/main/resources/application.properties[application.properties] file and
 objects filled with some demo data and persists them to the Cloud Spanner table created earlier.
 These POJOs are then retrieved and their contents are printed in the browser.
 
+.. To run the SpannerRepositoryExample:
+
+  mvn spring-boot:run -P spanner-repository-example
```

Comment:
I'm not a big fan of using profiles in this situation. We should rather control what runs based on a command-line argument passed to the main class. The is exactly what a `CommandLineRunner` is good for! :)
Alternatively, we can even split the sample into two modules, but I don't think that's worthwhile right now.

Reply:
Could you elaborate on why to not to use profiles? It seems like it could solve the problem here.

I don't think combining the two samples into a single main function is a good, because it would be a rather big sample program in that case and is contrary to the simplicity and the approachable feel of a sample.

I also don't think we should break this into two samples because nearly all of the config is shared between the two, and the repository even uses a template underneath.

Reply:
Having more than one main class in a Spring Boot application is not generally recommended. Even if you do insist on doing it here, the normal way to control which main class runs is to use `-Dstart-class`, not profiles.

Reply:
Done. Switched to use command line args + consolidate to 1 main. Spring seemed to have problems with having 2 main classes annotated with `@SpringBootApplication`; this may have been necessary.